.gitpod.yml

```diff
@@ -8,13 +8,10 @@ ports:
   # IDE
   - port: 8001
     onOpen: open-browser
-  # Live Reloading
```

Comment:
I think the first port should be `8000` instead of `8001`?

Reply:
Really good catch, thanks :) I'll update that 👍